content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -29,137 +29,130 @@ further_reading:
+      | sudo tar xzf - -C /opt/datadog
+      ```
+    - **Other distributions** - Download the tar archive with the glibc-linked binary:
+      ```bash
+      sudo mkdir -p /opt/datadog
+      curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>.tar.gz \
+      | sudo tar xzf - -C /opt/datadog
+      ```
+3. Add the [required environment variables](#required-environment-variables).
+
+4. Run the `/opt/datadog/createLogPath.sh` script, which creates a directory for the log files and sets appropriate directory permissions. The default directory for log files is `/var/log/datadog/dotnet`.
+
+5. Create application load.
+
+6. Visit [APM Live Traces][3].
+
+
+[1]: /agent/basic_agent_usage/
 [2]: https://en.wikipedia.org/wiki/Musl
+[3]: https://app.datadoghq.com/apm/traces
 {{% /tab %}}
 
 {{< /tabs >}}
 
-### Required Environment Variables
+By default, tracing is enabled in your `datadog.yaml` file under `apm_enabled: true` and the Agent listens for trace traffic at `localhost:8126`.
+
+### Required environment variables
 
 {{< tabs >}}
 
 {{% tab "Windows" %}}
 
-If your application runs in IIS, skip the rest of this section.
-
-For Windows applications **not** running in IIS, set these two environment variables before starting your application to enable automatic instrumentation:
+#### Applications not hosted in IIS
 
-**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
+To enable automatic instrumentation on For Windows applications not in IIS, you must set two environment variables before starting your application:
 
 Name                       | Value
 ---------------------------|------
 `CORECLR_ENABLE_PROFILING` | `1`
 `CORECLR_PROFILER`         | `{846F5F1C-F9AE-4B07-969E-05C26BC060D8}`
 
-#### Windows Services
+<div class="alert alert-warning"> 
+  <strong>Note:</strong> The .NET runtime tries to load a profiler into any .NET process started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. Don't set these environment variables globally because this causes all .NET processes on the host to load the profiler.
+</div>
 
-To automatically instrument a Windows Service, set the environment variables for that Service in the Windows Registry. Create a multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\<SERVICE NAME>` key. Then set the key's data to the values in the table:
-```text
-CORECLR_ENABLE_PROFILING=1
-CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
-```
+#### Windows services
```

Comment:
"Windows services" falls under "Applications not hosted in IIS" above. Should it use `#####`?
```suggestion
##### Windows services
```